### PR TITLE
Code updates

### PR DESCRIPTION
**Pull request by tomciopp**
@danielberkompas I'm replacing Poison with Jason, as the repo hasn't been touched in 2 years and it is slower.

### CODE REVIEW

**Comment by tomciopp** [comment]
@danielberkompas Any issues here?

File: .envrc [hunk 1]
```diff
@@ -0,0 +1 @@
+dotenv
```
Comment (danielberkompas) [inline .envrc:1]: Nitpick, can you remove this file? I don't want to assume all contributors will have `dotenv`.

Comment (tomciopp) [reply .envrc:1]: I have no problem removing the file (used 2 commits for a reason), but I don't think it should affect people who don't use direnv/dotenv.

Comment (tomciopp) [reply .envrc]: @danielberkompas Removed this file.

**Pull request merged**